src/components/families/FamilyDetailsSidebar.jsx

```diff
@@ -38,11 +61,27 @@ function FamilyDetailsSidebar({ isOpen, familyId, handleClose }) {
       open={isOpen}
       onClose={handleClose}
     >
-      <h3>
-        Hey {family?.parent?.first_name ?? ""} {family?.parent?.last_name ?? ""}
-        !
-      </h3>
+      <Typography variant="h3">
+        {family.parent.first_name} {family.parent.last_name}
+      </Typography>
+      <Divider variant="fullWidth" />
+      <Typography variant="h5" className="pb">
+        Basic Information
+      </Typography>
+      <DefaultInfo family={family} classes={classes} />
+      <ParentInfo family={family} classes={classes} />
     </Drawer>
+  ) : (
+    <Drawer
+      anchor="right"
+      variant="temporary"
+      className={classes.drawer}
+      classes={{
+        paper: classes.drawerPaper,
+      }}
+      open={isOpen}
+      onClose={handleClose}
+    />
```

Comment:
can we refactor this to reduce duplicate code? something like this could work!
```js
<Drawer [props]>
  { family ?
    <>
       // drawer contents here (typography, ParentInfo & DefaultInfo)
    </>
    : <></>  // empty fragment
  }
</Drawer>
```

Reply:
great catch! updated 8)